weave/tests/test_uploader.py

```diff
@@ -1453,3 +1456,88 @@ def test_upload_from_s3fs(test_basket):
         s3.rm(minio_path,recursive=True)
     if local_fs.exists(pantry_2.index.db_path):
         local_fs.rm(pantry_2.index.db_path)
+
+
+# Skip tests if pymongo is not installed.
+@pytest.mark.skipif(
+    "pymongo" not in sys.modules or not os.environ.get("MONGODB_HOST", False),
+    reason="Pymongo required for this test",
+)
+def test_upload_basket_mongo(test_basket):
+    """Testing pantry.upload_basket(), expected
+
+    to update the collections in mongodb
+    """
+    fs = test_basket.file_system
+    pantry_path = test_basket.pantry_path
+    mongo_client = get_mongo_db()
+
+    pantry = Pantry(IndexPandas,
+                    pantry_path=pantry_path,
+                    mongo_client=mongo_client,
+                    file_system=fs)
+
+    with tempfile.NamedTemporaryFile(delete=False) as temp_file:
+        uuid = pantry.upload_basket(
+            upload_items=[{'path':temp_file.name,'stub':False}],
+                basket_type="test-1",
+                metadata = {'Data Type':'text'}
+                ).values.tolist()[0][0]
+
+        temp_file.close()
+        os.unlink(temp_file.name)
+
+    collections = ("supplement", "metadata", "manifest")
+    mongo_db = mongo_client[pantry.pantry_path]
+    query = {'uuid': uuid}
+
+    for e in collections:
+        #Check, if document exist in the selected collection
+        assert uuid == mongo_db[e].find_one(query,{'_id':0,'uuid':1})['uuid']
+        #clean up, remove the document from the selected collection
```

Comment:
There should be a space after the # and the first letter of a comment should be capitalized, per the weave style guide. It might be worth just combining the two comments to say "# Check that the document exists in the selected collection, then remove the document."